stackanetes-services/compute/compute-node.yml

```diff
@@ -1,5 +1,6 @@
 type: daemonset
 name: compute-node
+label: compute-node
```

Comment:
Should we prefix them with `openstack-` to avoid label collision on big enterprise clusters?